generators/app/templates/src/components/ProviderWrapper/index.tsx

```diff
@@ -5,13 +5,13 @@ interface ActionType {
 }
 
 interface Props<U extends {}, V> {
-  Context: React.Context<any>;
+  context: React.Context<any>;
```

Comment:
I just changed this to `React.Context<{ state: U, dispatch: React.Dispatch<V> }>;` in the cookbook